### PR TITLE
Add OpsWorks support (stacks and custom layers)

### PR DESCRIPTION
**Pull request by DJRH**
Based on the separate pull request by @cmcarthur (thanks for your work so far). I've separated the resources as @dtan4 requested in https://github.com/dtan4/terraforming/pull/144 .

### CODE REVIEW

**Comment by dtan4** [comment]
Thank you for your work 👍 

I tested with sample Opsworks stack, and unfortunally `terraform plan` failed with generated files.

```bash
$ terraform plan
There are warnings and/or errors related to your configuration. Please
fix these before continuing.

Errors:

  * aws_opsworks_stack.My Sample Stack (Linux): %!s(MISSING): resource name can only contain letters, numbers, dashes, and underscores.My Sample Stack (Linux)
```

The solution is this: please normalize module name with [`normalize_module_name`](https://github.com/dtan4/terraforming/blob/48b6dc8da638464f4245221b655f7753ad62ff5b/lib/terraforming/util.rb#L12-L14) method beforehand.

```ruby
def module_name_of(stack)
  normalize_module_name(stack.name)
end
```

[ELB resource](https://github.com/dtan4/terraforming/blob/48b6dc8da638464f4245221b655f7753ad62ff5b/lib/terraforming/resource/elb.rb) uses the similar approach.

File: lib/terraforming.rb [hunk 1]
```diff
@@ -41,6 +41,8 @@
 require "terraforming/resource/internet_gateway"
 require "terraforming/resource/network_acl"
 require "terraforming/resource/network_interface"
+require "terraforming/resource/opsworks_stack"
+require "terraforming/resource/opsworks_custom_layer"
```
Comment (dtan4) [inline lib/terraforming.rb]: All resources are sorted alphanbetically. Please swap these resources.

File: lib/terraforming/cli.rb [hunk 2]
```diff
@@ -121,6 +121,16 @@ def nif
...
 
+    desc "opws", "OpsWorks Stack"
+    def opws
+      execute(Terraforming::Resource::OpsWorksStack, options)
+    end
+
+    desc "opwcl", "OpsWorks Custom Layer"
+    def opwcl
+      execute(Terraforming::Resource::OpsWorksCustomLayer, options)
+    end
```
Comment (dtan4) [inline lib/terraforming/cli.rb]: Please sort these alphabetically.

**Comment by DJRH** [comment]
Thanks for your comments @dtan4 - the issues should be resolved now 👍

**Comment by dtan4** [comment]
It works fine. LGTM 👍 Thanks ❗ 

```
aws_opsworks_custom_layer.My-Sample-Stack--Linux-_Node-js-App-Server: Refreshing state... (ID: f6e5fa2c-db8b-4a34-8ed4-798975a2f9f4)
aws_opsworks_stack.My-Sample-Stack--Linux-: Refreshing state... (ID: 492fdfd1-1d3c-431f-83cf-105816fff3d9)
```

Sorry for late response 🙇

**Pull request merged**